### ISSUE TITLE
[Request]: Lower resolution for Retina displays

### ISSUE THREAD

**Issue by frinkifail** [xCollateral/VulkanMod#512]
### Describe the request

On Retina displays, the game renders at a decently high resolution which is heavily demanding. An option to use lower resolutions (1400x900 as an example) would be a great addition to this mod.

**Comment by Felix14-v2** [comment]
Doesn't the fullscreen resolution option work on MacOS?

**Comment by Andarwinux** [comment]
This is also very useful on Windows. Not everyone uses fullscreen mode.

**Comment by frinkifail** [comment]
> Doesn't the fullscreen resolution option work on MacOS?

No, it does not. It only has 1 option which is the high resolution option.

**Comment by NXTler** [comment]
This seems more like a driver issue than a VulkanMod issue, as VulkanMod gets all the display/resolution information by the driver and creates an option for each.

**Comment by annpocoyo** [comment]
In macOS's native full screen (which is the trigger for macOS Game Mode), the full screen resolution option doesn't work.

The mod 'retiNO' works by setting the window hint `GLFW_COCOA_RETINA_FRAMEBUFFER` to false. A setting in VulkanMod to allow that to be done by the user would be appreciated.
